aws-rds-optiongroup/src/main/java/software/amazon/rds/optiongroup/ReadHandler.java

```diff
@@ -47,6 +49,7 @@ protected ProgressEvent<ResourceModel, CallbackContext> handleRequest(
                     final OptionGroup optionGroup = describeResponse.optionGroupsList().stream().findFirst().get();
                     final List<OptionConfiguration> optionConfigurations = Translator.translateOptionConfigurationsFromSdk(optionGroup.options());
                     final List<Tag> tags = listTags(proxyInvocation, optionGroup.optionGroupArn());
+                    context.setOptionGroupArn(optionGroup.optionGroupArn());
```

Comment:
As it is a part of ReadHandler, which is only invoked as a chain finalizer, I assume this arn set is a no-op. Could you please check it?

Reply:
I think I need some clarification offline about what exactly you are asking here.